examples/wfs.js

```diff
@@ -87,29 +82,27 @@ view.addLayer({
     },
 }, view.tileLayer);
 
-function colorFunctionBuildings(layer, node, featureCollection) {
-    var i;
-    var featureProperties;
-    var colors = [];
-
-    for (i = 0; i < featureCollection.features.length; i++) {
-        featureProperties = featureCollection.features[i].properties;
-
-        if (featureProperties.id.indexOf('bati_remarquable') === 0) {
-            colors.push(new itowns.THREE.Color(0x5555ff));
-        } else if (featureProperties.id.indexOf('bati_industriel') === 0) {
-            colors.push(new itowns.THREE.Color(0xff5555));
-        } else {
-            colors.push(new itowns.THREE.Color(0xeeeeee));
-        }
+function colorBuildings(properties) {
+    if (properties.id.indexOf('bati_remarquable') === 0) {
+        return new itowns.THREE.Color(0x5555ff);
+    } else if (properties.id.indexOf('bati_industriel') === 0) {
+        return new itowns.THREE.Color(0xff5555);
     }
+    return new itowns.THREE.Color(0xeeeeee);
+}
 
-    itowns.FeatureProcessing.assignColorsToFeatureCollection(
-        featureCollection, featureCollection.children[0], colors);
+function extrudeBuildings(properties) {
+    return properties.hauteur;
 }
 
 view.addLayer({
-    update: itowns.FeatureProcessing.update(colorFunctionBuildings),
+    type: 'geometry',
+    update: itowns.FeatureProcessing.update,
+    convert: itowns.Feature2Mesh.convert({
+        onMeshCreated: undefined,
```

Comment:
The `onMeshCreated` property could be a top level layer property: that's the way it's done for points and tiles.